lib/foreman/telemetry_sinks/prometheus_sink.rb

```diff
@@ -6,15 +6,15 @@ def initialize(opts = {})
     end
 
     def add_counter(name, description, instance_labels)
-      @prom.counter(name.to_sym, description)
+      @prom.counter(name.to_sym, docstring: description)
```

Comment:
@lzap should we pass the instance_labels? Prometheus client seems to have something called [pretty similarily](https://github.com/prometheus/client_ruby/blob/v1.0.0/lib/prometheus/client/registry.rb), but I have no clue if there is a reason we don't pass them on?